klaas/scripts/train_energy_regressor.py

```diff
@@ -53,7 +53,10 @@ def main(configuration_path, signal_path, predictions_path, model_path, key, ver
     regressor = eval(config['regressor'])
 
     log.info('Loading data')
-    df = read_data(file_path=signal_path, key=key)
+    df = read_data(file_path=signal_path, key=key,
+                   columns=training_variables.append(
+                       'MCorsikaEvtHeader.fTotalEnergy')
```

Comment:
I'm reluctant to hardcode feature names here.  Maybe we can use the target variable from the config file? This way it can be used with other projects as well. You know the ones with less insane variable names.